compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/native/ComposeLayer.jsNative.kt

```diff
@@ -63,10 +66,13 @@ internal class ComposeLayer(
         }
 
         override fun onPointerEvent(event: SkikoPointerEvent) {
-            if (supportsMultitouch) {
```

Comment:
it seems like if we are moving in this direction - I mean inspecting the nature of incoming event to resolve to multitouch this variable - supportsMultitouch is redundanant completely

Reply:
I dont know. Yes, here it's look redundant, but i fear to remove `internal expect value ` from project could to bloat the PR. Can we make this moment as a topic on separate PR as nice refactoring of the future?